src/api/AppIntent.ts

```diff
@@ -10,6 +10,8 @@ import { IntentMetadata } from './IntentMetadata';
  * An interface that relates an intent to apps
  */
 export interface AppIntent {
+  /** Details of the intet who's relationship to resolving applications is being described. */
```

Comment:
who's should be whose?

Reply:
You are correct and did a better job of spotting that than Grammarly, who even have a [blog article](https://www.grammarly.com/blog/whos-whose/#:~:text=Who's%20is%20a%20contraction%20linking,who's%20and%20whose%2C%20read%20on.) on how to pick the right one: